woeip/apps/air_quality/models.py

```diff
@@ -0,0 +1,73 @@
+from django.contrib.gis.db.models import LineStringField, PointField
+from django.contrib.postgres.fields import JSONField
+from django.db import models
+
+from woeip.apps.core.models import User
+
+
+class Route(models.Model):
+    """Named routes"""
+    name = models.CharField(max_length=256, unique=True)
+    path = LineStringField()
+
+    def __str__(self):
+        return self.name
+
+
+class Device(models.Model):
+    name = models.CharField(max_length=256)
+    manufacturer = models.CharField(max_length=256)
+    serial_number = models.CharField(max_length=256)
+    model_number = models.CharField(max_length=256)
+    calibration_date = models.DateField()
+    firmware_version = models.CharField(max_length=256)
+
+    def __str__(self):
+        return f"{self.name} {self.model_number} {self.serial_number}"
+
+
+class Sensor(models.Model):
+    """A sensor is something that measures something, i.e., it produces a single measurement value
+    at a time.
+    """
+    name = models.CharField(max_length=256)
+    unit = models.CharField(max_length=256,
+                            help_text="Measurement unit, e.g., mg/m3, ppm, etc.")
+    device = models.ForeignKey(Device, on_delete=models.SET_NULL, blank=True, null=True)
+
+    def __str__(self):
+        return f"{self.name} ({self.device.name})"
+
+
+class Session(models.Model):
+    """A single air quality outing. Can link to several SessionData, e.g., raw data files."""
+    date_collected = models.DateTimeField(auto_now_add=True)
+    route = models.ForeignKey(Route, on_delete=models.SET_NULL, blank=True, null=True)
+    collected_by = models.ForeignKey(User, on_delete=models.SET_NULL, blank=True, null=True)
+    metadata = JSONField(blank=True, null=True)
```

Comment:
I suggest removing this field for now. Add it later when there is a great need for it. If you add it now, you run the risk of it being an unstructured catch-all field that will become a mess to deal with.